src/contexts/swr/fetcher.ts

```diff
@@ -24,12 +23,11 @@ export default async function fetcher<ResponseData, RequestBody = {}>(
     requestOptions.method = 'POST';
     requestOptions.body = JSON.stringify(body);
   }
-
   const res = await fetch(`${baseurl}/glry/v1${path}`, requestOptions);
   const payload = await res.json();
 
-  if (isErrorResponse(payload)) {
-    throw new Error(payload.data.handler_error_user_msg);
+  if (!res.ok) {
```

Comment:
Is res.ok always built into 200 responses?

Reply:
Yup it should!
res.ok will be true if the response code is 2xx
Except on ie11 lol

https://developer.mozilla.org/en-US/docs/Web/API/Response/ok